docs/library/o2em.md

```diff
@@ -1,100 +1,50 @@
-
-<center> ![](..\image\core\all\load.png) </center>
-
-- Browse to the folder that contains the content you want to run.
-
-- Select the content that you want to run.
-
-<center> ![](..\image\core\all\screenshot_name.png) </center>
-
-- If you are asked which core to select, choose 'Odyssey2 / Videopac+ (O2EM)'.
-
-The content should now start running!
-
-### Authors
-
-- Daniel Boris|Andre de la Rocha
+- Daniel Boris
+- Andre de la Rocha
 - Arlindo M. de Oliveira
 
-## License
+The O2EM core is licensed under
+
+- [Artistic License](https://sourceforge.net/projects/o2em/)
 
 A summary of the licenses behind RetroArch and its cores have found [here](https://docs.libretro.com/tech/licenses/).
 
-The O2EM core is licensed under
+## BIOS
 
-- [Artistic License](https://sourceforge.net/projects/o2em/)
+Required or optional firmware files go in the frontend's system directory.
+
+| Filename   | Description                                      |              md5sum              |
+|:----------:|:------------------------------------------------:|:--------------------------------:|
+| o2rom.bin  | Odyssey2 BIOS - G7000 model - Required           | 562d5ebf9e030a40d6fabfc2f33139fd |
+| c52.bin    | Videopac+ French BIOS - G7000 model - Required   | f1071cdb0b6b10dde94d3bc8a6146387 |
+| g7400.bin  | Videopac+ European BIOS - G7400 model - Required | c500ff71236068e0dc0d0603d265ae76 |
+| jopac.bin  | Videopac+ French BIOS - G7400 model - Required   | 279008e4a0db2dc5f1c048853b033828 |
 
 ## Extensions
 
 Content that can be loaded by the O2EM core have the following file extensions:
 
 - .bin
 
-## Databases
-
 RetroArch database(s) that are associated with the O2EM core:
 
 - [Magnavox - Odyssey2](https://github.com/libretro/libretro-database/blob/master/rdb/Magnavox%20-%20Odyssey2.rdb)
 - [Phillips - Videopac+](https://github.com/libretro/libretro-database/blob/master/rdb/Philips%20-%20Videopac%2B.rdb)
 
-## BIOS
-
-Required or optional firmware files go in RetroArch's system directory.
-
-| Filename   | Description                                      |              md5sum              |
-|:----------:|:------------------------------------------------:|:--------------------------------:|
-| o2rom.bin  | Odyssey2 BIOS - G7000 model - Required           | 562d5ebf9e030a40d6fabfc2f33139fd |
-| c52.bin    | Videopac+ French BIOS - G7000 model - Required   | f1071cdb0b6b10dde94d3bc8a6146387 |
-| g7400.bin  | Videopac+ European BIOS - G7400 model - Required | c500ff71236068e0dc0d0603d265ae76 |
-| jopac.bin  | Videopac+ French BIOS - G7400 model - Required   | 279008e4a0db2dc5f1c048853b033828 |
-
 ## Features
 
-RetroArch-level settings or features that the O2EM core respects.
+Frontend-level settings or features that the O2EM core respects.
 
 | Feature           | Supported |
 |-------------------|:---------:|
 | Restart           | ✔         |
-| Screenshots       | ✔         |
 | Saves             | ✕         |

```

Comment:
I've just proposed a PR that adds support for save states.

Reply:
Added